doc/Addons.md

```diff
@@ -227,6 +227,7 @@ Called after the language detection. This can be used for alternative language d
 
 - **text**: The text that is analyzed.
 - **detected**: (input/output) Array of language codes detected in the related text.
```

Comment:
Please mention that it's an associative array with the language code as the keys and the language probability as the values.